src/modules/shared/components/ErrorPage/ErrorPage.module.scss

```diff
@@ -0,0 +1,82 @@
+@use "styles/abstracts" as *;
+
+$component: "c-error-page";
+
+.#{$component} {
+	height: 100%;
+	display: grid;
+	grid-template-rows: 2fr 1fr;
+	grid-template-columns: minmax($spacer-sm, 1fr) auto minmax($spacer-sm, 1fr);
+
+	&__title {
+		z-index: get-z-layer("foreground");
+		max-width: 63rem;
+		grid-row: 3;
+		color: white;
+		text-align: center;
+		margin: 0 auto;
+		padding: 0 $spacer-sm;
+		align-self: center;
+		font-size: 2.8rem;
+		line-height: $line-height-2xl;
+
+		@include respond-at("md") {
+			font-size: $font-size-4xl;
+		}
+	}
+
+	&__image-background {
+		background-color: $black;
+		grid-row: 1;
+		grid-column: 1 / 4;
+		display: grid;
+		grid-template-rows: 1fr 8rem 2fr;
+		justify-content: center;
+
+		@include respond-at("md") {
+			grid-template-rows: 1fr 15rem 2fr;
+		}
+	}
+
+	&__image-wrapper {
+		position: relative;
+		height: 100%;
+		width: 100vw;
+		max-width: 144rem;
```

Comment:
Persoonlijk zou ik dit op FullHD zetten omdat ik daar snel "not as designed"-commentaar op verwacht.
```suggestion
		max-width: 192rem;
```

Reply:
- In design is het tot 1440px
<img width="1248" alt="image" src="https://user-images.githubusercontent.com/85545258/163182316-5c7faec8-4521-44b1-a802-b1a5be7c3a80.png">
- In ticket staat tot 1440px https://meemoo.atlassian.net/browse/ARC-487
- Groter dan 1440 px zijn er clipping issues (wel op te lossen uiteraard)
<img width="1018" alt="image" src="https://user-images.githubusercontent.com/85545258/163182455-1cc86f49-21f4-47cc-974d-71f4ae093c20.png">